src/main/resources/application.properties

```diff
@@ -105,6 +105,10 @@ ccd.defaultPrintName=CCD Print
 ccd.defaultPrintDescription=Printing for CCD
 ccd.defaultPrintType=CCD Print Type
 
+am.role-assignment.pagination.enabled=${AM_ROLE_ASSIGNMENT_PAGINATION_ENABLED:false}
+am.role-assignment.starting.page.number=${AM_ROLE_ASSIGNMENT_STARTING_PAGE_NUMBER:0}
+am.role-assignment.page.size=${AM_ROLE_ASSIGNMENT_PAGE_SIZE:100}
```

Comment:
maybe we could group with some other, AC-related properties? (with similar prefixes)

Reply:
Regrouped